Library/.rubocop.yml

```diff
@@ -354,6 +350,15 @@ Style/HashAsLastArrayItem:
     - "/**/Formula/**/*.rb"
     - "**/Formula/**/*.rb"
 
+Style/InverseMethods:
+  Exclude:
+    # Core extensions are not available here:
+    - "Homebrew/standalone/init.rb"
+    - "Homebrew/rubocops/**/*"
```

Comment:
I'm guessing this also disables vanilla Ruby InverseMethods checking.

If so, maybe worth having a `InverseMethodsExtension` cop that is nothing but a subclass of `RuboCop::Cop::Style::InverseMethods`, but will I think allow us to separate configs.

Reply:
Thanks for the call out, I should have added a bit more on my thought process. Let's enumerate some options here, because there are several. More or less in my order of preference, they include:

- Add `:exclude?: :include?` here, but exclude `Style/InverseMethods` cop where `exclude?` is unavailable (what I've done above)
- Omit `:exclude?: :include?` from the cop, as we do with `Style/InvertibleUnlessCondition` below
- Add `:exclude?: :include?` to the cop, use inline disable comments where `exclude?` is unavailable
- Just import `extend/string` and/or `extend/array` anywhere there's a violation, once some version of https://github.com/Homebrew/brew/pull/16463 lands
- Omit `:exclude?: :include?` from the cop, vendor the `Rails/NegateInclude` include cop (perhaps a simpler, though less extensible, version of what you're suggesting, IIUC)
- Add an `InverseMethodsExtension` cop that is nothing but a subclass of `RuboCop::Cop::Style::InverseMethods`, but would allow us to separate configs
- Remove the ActiveSupport [dependency](https://github.com/rubocop/rubocop-rails/blob/master/lib/rubocop-rails.rb#L5-L6) in rubocop-rails (a long shot, but hey)

I'm generally biased toward whichever results in the least amount of add'l code/comments, and I'm pretty ok if we don't have 100% coverage of `InverseMethods`. That said, I'm happy to do any of these (except the last one 🙈) if there is consensus otherwise. Let me know if this is helpful/influences where you think we should go with this.